doc/visual-programming/source/widgets/classify/randomforest.rst

```diff
@@ -1,9 +1,9 @@
-Random Forest
-=============
+Random Forest Classification
+============================
 
 .. figure:: icons/random-forest.png
 
-Random forest learner
+Random forest classification
 
```

Comment:
All the variants are fine. Learner is shorter, but "learning algorithm" more common in ML.